webpack/assets/javascripts/react_app/components/common/Alert.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
i feel like maybe this should be a functional component, without a state. parent should render it or not